### PR TITLE
BXMSPROD-1086 sonar analysis for PR only

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

**JIRA**: https://issues.redhat.com/browse/BXMSPROD-1086

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by Ginxo** [comment]
test job: https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/compile/job/drools-master.compile/276

**Review by rsynek — APPROVED** [review]
LGTM, thank you!

**Comment by rsynek** [comment]
@Ginxo I noticed the profile "run-code-coverage" being used on several lines in the  .ci/pullrequest.properties. Does it activate for CDB too? If so, it should be removed as it just prolongs the build. Similarly to the profile "sonarcloud-analysis", it should be activated only for normal PR checks.

**Review by mbiarnes — APPROVED** [review]
+1

File: .ci/pullrequest.properties [hunk 1]
```diff
@@ -5,7 +5,6 @@ goals.appformer.upstream=-e clean install -DskipTests -Dgwt.compiler.skip=true -
 goals.drools.upstream=-e clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true
 goals.kie-wb-common.upstream=-e clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true
 goals.optaplanner.upstream=-e clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true
-goals.default.sonarcloud=-nsu generate-resources -Psonarcloud-analysis
```
Comment (mareknovotny) [inline .ci/pullrequest.properties:8]: thanks

**Comment by Ginxo** [comment]
> @Ginxo I noticed the profile "run-code-coverage" being used on several lines in the .ci/pullrequest.properties. Does it activate for CDB too? If so, it should be removed as it just prolongs the build. Similarly to the profile "sonarcloud-analysis", it should be activated only for normal PR checks.

@rsynek .ci/pullrequest.properties is only used from pull request jobs https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/582a96a7ffcc1597c6ea3e8973fd82d8340fb149/.ci/pullrequest.stages#L5

**Pull request merged**